tonytanaka/index.js

```diff
@@ -0,0 +1,104 @@
+// 1 Person Class
+class Person {
+    constructor (name, age) {
+        this.personName = name;
+        this.personAge = age
+    }
+    render(){
+        return `${this.personName}, ${this.personAge} years old`
+    }
+}
+const friend = new Person('Israel', 64)
+console.log(friend.render())
+
+// 2 Volume
+class Volume {
+    constructor (r, h) {
+        this.radius = r;
+        this.height = h;
+    }
+    calculate() {
+        return Math.PI * this.radius ** (2 * this.height)
+    }
+}
+const cylinder = new Volume (2, 3)
```

Comment:
Task 2 ==> Volume of a cylinder : V = πr^2h - r  ==> The output is 201.06192982974676 but it must be  37.6991 
Therefore instead of 
return Math.PI * this.radius ** (2 * this.height) 
you have to return     
return (Math.PI * Math.pow(this.radius, 2) * this.height).toFixed(4);
http://mathbydesign.thinkport.org/images/pdfs/Hint_VolumeofCylinder.pdf